src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1706,6 +1712,82 @@ IOperation negateNullable(IOperation operand)
             }
         }
 
+        private IOperation VisitUserDefinedBinaryConditionalOperator(IBinaryOperation binOp, int? captureIdForResult)
+        {
+            SpillEvalStack();
+
+            Compilation compilation = ((Operation)binOp).SemanticModel.Compilation;
```

Comment:
Probably worth saving into a field, as we are using it for multiple nodes. Can be done in a separate PR.

Reply:
> Probably worth saving into a field, as we are using it for multiple nodes. Can be done in a separate PR.

There is a PROTOTYPE comment about this at the beginning of the file

---
In reply to: [190737665](https://github.com/dotnet/roslyn/pull/27070#discussion_r190737665) [](ancestors = 190737665)

Reply:
Ah, thanks.